src/styles/main.scss

```diff
@@ -1,3 +1,184 @@
-body {
+$main-color: #00acdc;
+
+* {
+  box-sizing: border-box;
+}
+
+html {
+  font-family: "Roboto", sans-serif;
+  font-weight: 400;
+  font-size: 11px;
   margin: 0;
 }
+
+body {
+  font-family: Roboto, sans-serif;
+  font-style: 500;
+  max-width: 1200px;
+  margin: 0 auto;
+}
+
+.main {
+  padding: 0 40px;
+}
+
+.section {
+  display: flex;
+  flex-direction: row;
+  flex-wrap: wrap;
+  justify-content: space-between;
+
+  max-width: 944px;
+  margin: 0 auto;
+}
+
+/* #region header */
+.header {
+  display: flex;
+  align-items: center;
+  justify-content: space-between;
+  padding: 0 50px;
+  box-shadow: 0 2px 4px rgba(0, 0, 0, 0.05);
+  margin-bottom: 50px;
+}
+
+.logo {
+  width: 40px;
+  height: 40px;
+}
+
+.nav {
+  &__list {
+    display: flex;
+    padding: 0;
+    margin: 0;
+  }
+
+  &__link {
+    font-size: 12px;
+    text-decoration: none;
+    color: #000;
+    text-transform: uppercase;
+    font-weight: 500;
+  }
+
+  &__item {
+    display: flex;
+    position: relative;
+    list-style: none;
+    margin-left: 20px;
+    line-height: 60px;
+    height: 60px;
+  }
+
+  &__item:first-child {
+    margin-left: 0;
+  }
+  
+  &__link:hover,
+  .is-active {
+    color: #{$main-color};
+  }
+}
+
+.is-active::after {
+  content: "";
+  position: absolute;
+  right: 0;
+  bottom: 0;
+  left: 0;
+  height: 4px;
+  background-color: #{$main-color};
```

Comment:
I think you should to separate the variable "main-color" to two variables. First variable for the header and second for the button. After all, the design of the header can change and design of the button isn't. Or vice versa.